src/Components/Todo/index.jsx

```diff
@@ -35,26 +36,47 @@ const Todo = () => {
   const [incomplete, setIncomplete] = useState([]);
   const { handleChange, handleSubmit } = useForm(addItem, defaultValues);
 
-  function addItem(item) {
-    item.id = uuid();
+  async function addItem(item) {
+
+    // item.id = uuid();
     item.complete = false;
-    console.log(item);
+    const config = {
+      baseURL: 'https://api-js401.herokuapp.com/api/v1/todo',
+      method: 'post',
+      data: item
+    };
+    let response = axios(config);
+    console.log('item', response.data);
     setList([...list, item]);
```

Comment:
Missing an `await`, which means you are adding a promise to your list instead of an item. Also, you may want some error handling in case something happens with this or any other axios requests. If done well, this will also help keep your state and the database in sync without having to make extra requests.